english/count-the-marshmellows/eat-all-the-marshmellows/get-breakfast/get-breakfast.md

```diff
@@ -1,7 +1,14 @@
-HECK YES. You found the best ending. (Breakfast is always the best ending.)
```

Comment:
@lindapescatore, I'm sorry this is so difficult. If you look at the line numbers on the left, you can see that this paragraph used to be on lines 1 through 5, but now it is on line 1 only. Now this might not seem like a big problem, since it still displays okay on GitHub, but things like `git diff` output will be a lot harder to read if each line is too long, since most Git tools don't auto-wrap lines.

My advice on how to fix this is to figure out how to get your text editor to display line numbers (if it's not already). If you use Sublime, I think it should display them by default, otherwise you may have to Google. Then, you can use enter to make the line numbers match what they were before.

If that doesn't work, you could always check out a new branch based off the master branch (which would still have the paragraph), then copy and paste your changes into the new branch.

Again, I'm sorry this is such a pain. I know that learning to fix whitespace problems can be very difficult, especially since the difference is not always visible. I hope you are able to get it fixed.